packages/react/package.json

```diff
@@ -43,6 +43,7 @@
   },
   "dependencies": {
     "@primer/behaviors": "^1.3.4",
+    "@oddbird/popover-polyfill": "^0.3.1",
```

Comment:
Looks like the latest version is `0.4.0`. Could we use that version for this initial ship please?